app/src/main/java/net/bible/android/view/activity/readingplan/DailyReading.java

```diff
@@ -126,9 +126,19 @@ public void onCreate(Bundle savedInstanceState) {
 	        for (int i=0; i<mReadings.getNumReadings(); i++) {
 	        	final int readingNo = i;
 	            View child = getLayoutInflater().inflate(R.layout.reading_plan_one_reading, null);
-	
-	            // Ticks
-	            mImageTickList.add((ImageView)child.findViewById(R.id.tick));
+
+				// Ticks
+				ImageView mImageTick = (ImageView)child.findViewById(R.id.tick);
+				mImageTickList.add(mImageTick);
+				// Allow check box to be clicked to mark off the day
+				mImageTick.setOnClickListener(new OnClickListener() {
+					@Override
+					public void onClick(View view) {
+						ReadingStatus status = readingPlanControl.getReadingStatus(mDay);
+						status.setRead(readingNo);
+						updateTicksAndDone();
+					}
+				});
```

Comment:
Hmm.. This pull request reveals that there's still some mixed tabs/spaces in the code. I need to check this. Problem is not in your code but old codebase.

Reply:
I think I saw a note in Git somewhere about changing LF line endings to CRLF. I don't find that note now. I don't know if it would have anything to do with that.

Reply:
I see now what you mean. There are a mixture of spaces and tabs in this file. It is very easy to fix in Android studio. I can resubmit a pull request with the fix if you want. Are you using spaces?

Reply:
I think it's better that you do fix that yourself in this pull request as your pull request is OK (i.e. tabs are used consistently).

I'll go through all codebase over again with some automated method as a separate task. (I did it already once, but for some reason I have missed some...)

Reply:
Sounds good. For future do I need to have my editor defaulting to tabs or spaces? Or does it matter?

Reply:
I'm using .editorconfig, and Android Studio obeys it, so you should be set up OK.